### ISSUE TITLE
[BUG] Previous developer accounts soft-locked by SideStore

### ISSUE THREAD

**Issue by GreyAsteroid** [SideStore/SideStore#410]
### Describe the bug

Enrolling in the Developer Program removes the sideloading limits for an account even after your membership has expired. Because of this App IDs do not actually expire so for example I currently have 62 App IDs on my account, some from multiple years ago. The issue is that SideStore sees that I'm a "Free Developer Account" and assumes that I have the 10 App ID limit and shows "0 App IDs Remaining" and seems to make that check before any install so that I am unable to install any applications beside SideStore itself. If I install applications manually using AltServer (not SideServer since it revokes my certificate every time) then I'm able to have an infinite amount of applications installed at the same time with the 365 day expiry.

### Instructions to reproduce

1. Become a member of Apple's Developer Program.
2. Let your membership lapse.
3. Use more App IDs than what is available with a free acount.
4. You're now locked from using SideStore.

### What version of SideStore are you using?

SideStore 0.4.0

**Comment by Nythepegasus** [comment]
Could you try using this build here: [SideStore-0.4.0-nightly.2023.07.11.125bf8bd.ipa.zip](https://github.com/SideStore/SideStore/files/12034037/SideStore-0.4.0-nightly.2023.07.11.125bf8bd.ipa.zip)

**Comment by GreyAsteroid** [comment]
> Could you try using this build here: [SideStore-0.4.0-nightly.2023.07.11.125bf8bd.ipa.zip](https://github.com/SideStore/SideStore/files/12034037/SideStore-0.4.0-nightly.2023.07.11.125bf8bd.ipa.zip)

Issue still persists with this build, I recieve the "You may only register 10 App IDs every 7 days" pop up.

**Comment by Nythepegasus** [comment]
We will have to look into this more, thank you for reporting!

**Comment by GreyAsteroid** [comment]
I'll go ahead and close this once https://github.com/SideStore/SideStore/pull/508 is merged.

**Comment by powerhousepr** [comment]
Welp, this is still a thing. I became an App developer and Side store wont even let me use my Apple iD anymore :/

**Comment by LiuJiangshan** [comment]
> Welp, this is still a thing. I became an App developer and Side store wont even let me use my Apple iD anymore :/


### Solution:
1. Use the desktop version of **SideStore** to install the `.ipa` file, which will automatically generate a new App ID.
2. After installation, uninstall the app, then use the **mobile version of SideStore** to install the same `.ipa` file.
3. This bypasses the "You may only register 10 App ID's every 7 days" limitation.

**Comment by Tratosca** [comment]
Can confirm. Previously on a paid developer. Latest nightly still can't allow to sideload anything due to Apps IDs (which won't be automatically deleted ever).

**Comment by GrPK** [comment]
Still a thing here as well///

**Comment by Dyvinia** [comment]
i have this issue

**Comment by duronald** [comment]
> i have this issue

your best bet is to fork it and build it in GitHub actions. im not sure if a patch will be merged into mainline anytime soon. the patch is luckily small and quick
